platform/android/export/gradle_export_util.h

```diff
@@ -94,8 +101,45 @@ Error store_string_at_path(const String &p_path, const String &p_data) {
 // This method will be called ONLY when custom build is enabled.
 Error rename_and_store_file_in_gradle_project(void *p_userdata, const String &p_path, const Vector<uint8_t> &p_data, int p_file, int p_total) {
 	String dst_path = p_path.replace_first("res://", "res://android/build/assets/");
-	Error err = store_file_at_path(dst_path, p_data, Z_NO_COMPRESSION);
+	Error err = store_file_at_path(dst_path, p_data);
 	return err;
 }
 
+// Creates strings.xml files inside the gradle project for different locales.
+Error _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset, const String &project_name) {
+	// Stores the string into the defualt values directory.
+	String processed_default_xml_string = vformat(base_xml_string, project_name.xml_escape(true));
+	store_string_at_path("res://android/build/res/values/godot_project_name_string.xml", processed_default_xml_string);
+
+	// Searches the Gradle project res/ directory to find all supported locales
+	DirAccessRef da = DirAccess::open("res://android/build/res");
+	if (!da) {
+		return ERR_CANT_OPEN;
+	}
+	da->list_dir_begin();
+	while (true) {
+		String file = da->get_next();
+		if (file == "") {
+			break;
+		}
+		if (!file.begins_with("values-")) {
+			// TODO: This code assumes all "values-" directories are for localization.
```

Comment:
Nit: as written, this is more of a "Note" than a TODO. (A TODO would indicate a change you plan to make.)